examples/apollo-fullstack/src/fullstack-typegen.ts

```diff
@@ -196,7 +196,7 @@ export interface NexusGenArgTypes {
   }
 }
 
-export interface NexusGenAbstractResolveReturnTypes {}
+export interface NexusGenAbstractTypesMapResolveTypeMethodReturnType {}
```

Comment:
Not sure I like this naming any better, it's super hard to read

Reply:
Do you feel one of these makes it easier or harder to understand the intent?

Reply:
I think it makes it harder to understand the intent. How about something like `NexusGenAbstractTypeMembers` ?

Reply:
I think it'll be hard to find the perfect thing so fine with me. 

I'm not actually sure we wanted to encode that these types are the method return type or not come to think of it... 🤔 

If not your name idea makes sense especially.

I'm not sure we care about encoding the fact that this is a lookup/map/whatever data structure to go from an abstract type to its members. It might be useful to have a naming pattern for all such data structures. E.g.: `<A>To<B>` or `<A>Map<B>`. Looking at the dozen or two Nexus gen fields it feels some naming conventions could help.